components/PageTitle/PageTitle.module.scss

```diff
@@ -1,10 +1,4 @@
-.background {
-    @include full-width;
-}
-
 .container {
-    @include container;
-
     padding: $spacing-8 0 0;
 
     @include mobile-only {
```

Comment:
This looks wrong:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/1641218/180217070-c71c245e-5bdd-49be-a339-6771b2de4ad8.png">

Let's simplify both padding rules on lines 2 and 5 to only specify `padding-top`.

Reply:
Sure, on it